examples/simple_connect_test.py

```diff
@@ -67,6 +68,7 @@ def setup_class(self):
       self.ad.load_snippet('mbs', 'com.google.android.mobly.snippet.bundled')
     except Exception:
       logging.error(_MESSAGE_MOBLY_SNIPPET_BUNDLE_IS_NOT_INSTALLED)
```

Comment:
Should we use `logging.exception` to log the full traceback of the exception as well?

Reply:
Now the exception is re-raised so the full traceback will be printed. Is there any advantange that would make us want to log the full traceback as well?

Reply:
Got it, saw `raise` below.